s3-private/iam.tf

```diff
@@ -0,0 +1,53 @@
+resource "aws_iam_group" "usage" {
+  name = "${var.project}-${var.environment}-s3-usage"
+}
+
+resource "aws_iam_policy" "usage" {
+  name        = "${var.project}-${var.environment}-s3-usage"
+  description = "For uploading/downloading encrypted storage files on S3"
+
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:ListAllMyBuckets",
+        "s3:GetBucketLocation"
+      ],
+      "Resource": "arn:aws:s3:::*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:DeleteObject",
```

Comment:
Did we intentionally not add read/wrote to object versions?

Reply:
Knew there was something missed off! We aren't currently using versions for the first example on uploads.